internal/pkg/cli/svc_delete.go

```diff
@@ -36,10 +36,10 @@ const (
 
 const (
 	fmtSvcDeleteStart             = "Deleting service %s from environment %s."
-	fmtSvcDeleteFailed            = "Failed to delete service %s from environment %s: %v."
-	fmtSvcDeleteComplete          = "Deleted service %s from environment %s."
-	fmtSvcDeleteResourcesStart    = "Deleting resources of service %s from application %s."
-	fmtSvcDeleteResourcesComplete = "Deleted resources of service %s from application %s."
+	fmtSvcDeleteFailed            = "Failed to delete service %s from environment %s: %v.\n"
+	fmtSvcDeleteComplete          = "Deleted service %s from environment %s.\n"
+	fmtSvcDeleteResourcesStart    = "Deleting resources of service %s from application %s.\n"
```

Comment:
Should we remove the new line from this start?

Reply:
That's the one that's currently being used with a Stop. I am going to make a SvcDeleteResourcesFailed (and the equivalent for `job_delete`) in a separate PR and will adjust newlines accordingly.